demisto_sdk/dev_tools/lint_manager.py

```diff
@@ -236,7 +236,7 @@ def create_failed_unittests_file(self, failed_unittests):
         The file will be read in slack_notifier script - which will send the failed unittests to the content-team
         channel.
         """
-        with open('./Tests/failed_unittests.txt', "w") as failed_unittests_file:
+        with open('./artifacts/failed_unittests.txt', "w") as failed_unittests_file:
```

Comment:
this is what we read in the Content build?

Reply:
We will use the artifact url in container 0 (content repo). This I need to put this in artifacts.